src/App.js

```diff
@@ -85,11 +60,49 @@ function SignIn() {
   );
 }
 
+function App() {
+  window.addEventListener("load", (event) => auth.signOut());
```

Comment:
Never register event handlers at the component level in react since this will get called on every single render and cause a memory leak. Use useEffect.